src/open_producten/utils/serializers.py

```diff
@@ -11,7 +12,11 @@ def build_array_duplicates_error_message(objects: list, field: str, errors):
     for idx, obj in enumerate(objects):
         if obj in object_set:
             errors_messages.append(
-                f"Duplicate {type(obj).__name__} id: {obj.id} at index {idx}"
+                _(
+                    "Dubbel {} id: {} op index {}.".format(
+                        type(obj).__name__, obj.id, idx
+                    )
+                )
```

Comment:
```suggestion
                _(
                    "Dubbel {} id: {} op index {}."
                ).format(
                        type(obj).__name__, obj.id, idx
                    )
```

Format should be outside of the translation. I think otherwise it creates a translation for every formatted string.
Although idk if it will ever actually be translated